crates/l2/Makefile

```diff
@@ -1,10 +1,11 @@
 .DEFAULT_GOAL := help
 
-.PHONY: help init down clean init-local-l1 down-local-l1 clean-local-l1 init-l2 down-l2 deploy-l1 deploy-block-executor deploy-inbox setup-prover test ci_test update-cli-contracts
```

Comment:
Keep `.PHONY` at the top.

Reply:
The Makefile has many targets and some of them weren't included in the `.PHONY`, that's why i've changed the location to each "subsection" of the Makefile, because it's easier to keep track of each target so they are included.

I will put it back at the top

Reply:
Done in [c7aa3b7](https://github.com/lambdaclass/ethrex/pull/1466/commits/c7aa3b7938549869411b412f53caf64eac0bbb2e)